### PR TITLE
fix(docs): replace broken links

### LINKED ISSUE #329: Documentation broken link, page not found error
**Describe the bug**
Some links are broken

**To Reproduce**
1. Go to https://docs.accordproject.org/docs/logic-ergo.html
2. Click on the Accord Project Template Specification hyperlink
3. A page not found is returned

**Expected behavior**
The link should redrect to the according page

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version 87.0.4280.66

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

# Closes #329 #328 #344 <CORRESPONDING ISSUE NUMBER>

### Changes

- Fixed Broken hyperlink<ONE>

### Related Issues
- Issue #328  #344<NUMBER>
- Pull Request #<NUMBER>

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

**Review by irmerk — CHANGES_REQUESTED** [review]
It looks like the link is still broken, and should be pointing to https://docs.accordproject.org/docs/accordproject-template.html

**Comment by Aniruddha-Shriwant** [comment]
@irmerk 
as @dselman mention [here](https://github.com/accordproject/techdocs/issues/329#issuecomment-782578647)

> clicking on the 0.21 version link in the header, and then selecting "unstable > Documentation"

then it will work...

**Comment by irmerk** [comment]
Ah yes, my mistake. @jeromesimeon @dselman can you review this, I'm not sure if the pointer in this PR (https://docs.accordproject.org/docs/ref-ergo-spec.html) works or if it should be pointing to https://docs.accordproject.org/docs/accordproject-template.html

**Comment by Aniruddha-Shriwant** [comment]
> Ah yes, my mistake. @jeromesimeon @dselman can you review this, I'm not sure if the pointer in this PR (https://docs.accordproject.org/docs/ref-ergo-spec.html) works or if it should be pointing to https://docs.accordproject.org/docs/accordproject-template.html

Well my apologizes, I was also not sure about this but I thought that there was a keyword `Accord Project Template "Specification"` and I had seen this (https://docs.accordproject.org/docs/ref-ergo-spec.html) with the heading "Specification" under Reference Manual in Ergo Reference, hence commented [this](https://github.com/accordproject/techdocs/issues/329#issuecomment-782565494)

File: docs/accordproject-template.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ TemplateMark is the markup format in which the text for Accord Project templates
 
 It also provides the ability to specify the document structure (e.g., headings, lists), to highlight certain terms (e.g., in bold or italics), to indicate text which is optional in the agreement, and more.
```
Comment (Aniruddha-Shriwant) [inline docs/accordproject-template.md:81]: As a beginner, I was going through the documentation and found that there was a typo in **Markdown Text** hyperlink, hence added a commit to fix it...

File: docs/accordproject-template.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ TemplateMark is the markup format in which the text for Accord Project templates
...
 
-_More information about the Accord Project markup can be found in the [Markdown Text](markup-template) Section of this documentation._
+_More information about the Accord Project markup can be found in the [Markdown Text](markup-templatemark) Section of this documentation._
```
Comment (Aniruddha-Shriwant) [inline docs/accordproject-template.md:83]: As a beginner, I was going through the documentation and found that there was a typo in the **Markdown Text** hyperlink, hence added a commit to fix it...

File: docs/logic-ergo.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Ergo aims to:
 - ensure safe execution: the language should prevent run-time errors and non-terminating logic
 - be blockchain neutral: the same contract logic can be executed either on and off chain on a variety of distributed ledger technologies
 - be formally specified: the meaning of contracts should be well defined so it can be verified, and preserved during execution
-- be consistent with the [Accord Project Template Specification](accordproject-specification)
+- be consistent with the [Accord Project Template Specification](ref-ergo-spec)
```
Comment (dselman) [inline docs/logic-ergo.md:15]: Can you please change this to link to:
https://docs.accordproject.org/docs/accordproject-template.html

And modify the text to be

- be consistent with the Accord Project templates

**Review by dselman — CHANGES_REQUESTED** [review]
One small change, but looks good.

File: docs/logic-ergo.md (hunk 3 above)
Comment (Aniruddha-Shriwant) [reply docs/logic-ergo.md:15]: Ok sure :)

**Review by irmerk — CHANGES_REQUESTED** [review]
You'll want to rebase to resolve the conflicting files.

**Comment by Aniruddha-Shriwant** [comment]
Well, what I was having the fear of had happened now :)
Those merge conflicts :/ 
I think they had caused due to changing the same part of the files...will try to resolve them

**Comment by Aniruddha-Shriwant** [comment]
Well, @irmerk wait :) 
Sorry for this but I messed up in rebasing in vscode hence tries to resolve those commits on GitHub but then I saw that files changed were only showing 2 files changed but before this mess it was 3 :) 
#328's file should be changed :) 
I had seen this the first time and messed up sorry for that :(
Should I again change the file to add commit for #328?

**Comment by irmerk** [comment]
You could make these changes in a clean branch and close this one off and open a new PR?

**Comment by Aniruddha-Shriwant** [comment]
Yeah @irmerk that sounds perfect 😅, I will make a new PR for this.

**Comment by Aniruddha-Shriwant** [comment]
> but then I saw that files changed were only showing 2 files changed but before this mess, it was 3 

@irmerk I tried to figure out why that happened, so it was due to [this](https://github.com/accordproject/techdocs/pull/337) PR as this PR gets merged it had fixed #328 and when I rebase my branch as the file was already changed, hence it showed 2 files changed instead of 3 
I saw this at the time of creating new PR, else this PR was good to merge without any issues :)
I had created new PR with the same changes [here](https://github.com/accordproject/techdocs/pull/345)

**Pull request closed without merging**